<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFEFCSSimScriptTab</class>
 <widget class="QWidget" name="QFEFCSSimScriptTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QVisibleHandleSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="grpScript">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string> Script </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="labFile">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFCompleterTextEdit" name="edtScript" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpSim">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string> Simulation Output: </string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="1" column="0" colspan="4">
        <widget class="QtLogFile" name="pteOutput" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labETA">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnImport">
         <property name="text">
          <string>import results ...</string>
         </property>
         <property name="icon">
          <iconset resource="../fcs/qfrdrfcs.qrc">
           <normaloff>:/fcs_rawdata.png</normaloff>:/fcs_rawdata.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFCompleterTextEdit</class>
   <extends>QWidget</extends>
   <header>qfcompletertextedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtLogFile</class>
   <extends>QWidget</extends>
   <header>qtlogfile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../fcs/qfrdrfcs.qrc"/>
 </resources>
 <connections/>
</ui>
